src/v2/guide/single-file-components.md

```diff
@@ -1,43 +1,45 @@
-- **No CSS support** means that while HTML and JavaScript are modularized into components, CSS is conspicuously left out
-- **No build step** restricts us to HTML and ES5 JavaScript, rather than preprocessors like Pug (formerly Jade) and Babel
+- **Les définitions globales** forcent à avoir un nom unique pour chaque composant
+- **Les templates sous forme de chaînes de caractères** ne bénéficient pas de la coloration syntaxique et requièrent l'usage de slashes disgracieux pour le HTML multiligne.
+- **L'absence de support pour le CSS** signifie que le CSS ne peut pas être modularisé comme HTML et JavaScript
+- **L'absence d'étape de build** nous restreint au HTML et à JavaScript ES5, sans pouvoir utiliser des préprocesseurs tels que Babel ou Pug (anciennement Jade).
 
-All of these are solved by **single-file components** with a `.vue` extension, made possible with build tools such as Webpack or Browserify.
 
-Here's a simple example of a file we'll call `Hello.vue`:
+Tous ces désavantages sont résolus par les **composants monofichiers** avec une extension `.vue`, rendus possibles par les outils de build tels que Webpack ou Browserify.
+
+Voici un exemple simple de fichier que nous appellerons `Hello.vue` :
 
 <img src="/images/vue-component.png" style="display: block; margin: 30px auto">
 
-Now we get:
+Maintenant nous avons :
 
-- [Complete syntax highlighting](https://github.com/vuejs/awesome-vue#syntax-highlighting)
-- [CommonJS modules](https://webpack.github.io/docs/commonjs.html)
-- [Component-scoped CSS](https://github.com/vuejs/vue-loader/blob/master/docs/en/features/scoped-css.md)
+- [Une coloration syntaxique complète](https://github.com/vuejs/awesome-vue#syntax-highlighting)
+- [Des modules CommonJS](https://webpack.github.io/docs/commonjs.html)
+- [Du CSS à la portée limitée au composant](https://github.com/vuejs/vue-loader/blob/master/docs/en/features/scoped-css.md)
 
-As promised, we can also use preprocessors such as Jade, Babel (with ES2015 modules), and Stylus for cleaner and more feature-rich components.
+Et comme promis, nous pouvons aussi utiliser des préprocesseurs tels que Pug, Babel (avec les modules ES2015), et Stylus pour obtenir des composants plus lisibles et plus riches en fonctionnalités.
 
 <img src="/images/vue-component-with-preprocessors.png" style="display: block; margin: 30px auto">
 
-These specific languages are just examples. You could just as easily use Bublé, TypeScript, SCSS, PostCSS - or whatever other preprocessors that help you be productive. If using Webpack with `vue-loader`, it also has first-class support for CSS Modules.
+Ces langages spécifiques ne sont que des exemples; vous pourriez tout aussi aisément utiliser Bublé, Typescript, SCSS, PostCSS - ou tout autre préprocesseur qui vous aide à être productif. Si vous utilisez Webpack avec `vue-loader`, vous aurez aussi un outil de premier choix pour les modules CSS.
+
+### Qu'en est-il de la séparation des préoccupations (Separation of concerns) ?
```

Comment:
Salut ! je suis d'accord que le mot préoccupations est un peu malheureux. Attention, "concern" ne se limite pas au langage, et on peut très bien avoir une "separation of concerns" au sein d'un même langage ( https://en.wikipedia.org/wiki/Separation_of_concerns ) . Toutefois séparation des rôles n'est jamais utilisé, séparations des responsabilités presque jamais (c'était ma proposition) contrairement à séparation des préoccupations; c'était l'unique motif de notre choix; du coup je pensais qu'on restait là dessus. 

(pour ma part je trouve que "séparation des responsabilités" est le plus adapté en terme de sens strict et aussi parce qu'on trouve parfois ce mot échangés avec concern, comme ici : https://en.wikipedia.org/wiki/Single_responsibility_principle )